index.html

```diff
@@ -0,0 +1,213 @@
+        </div>
+        <div class="Card3">
+          <img src="img/Snapshoot Portfolio (2).png" alt="" />
+          <div class="Card3-P">
+            <h4>Tonic</h4>
+            <div class="Card3-tech">
+              <p class="P1">Canopy</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P2">Back End Dev</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P3">2015</p>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
+        <div class="Card4">
+          <img src="img/Snapshoot Portfolio (3).png" alt="" />
+          <div class="Card4-P">
+            <h4>Multi-Post Stories</h4>
+            <div class="Card4-tech">
+              <p class="P1">Canopy</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P2">Back End Dev</p>
+              <img src="img/Counter.png" alt="" />
+              <p class="P3">2015</p>
+            </div>
+          </div>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul>
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+          </ul>
+          <button type="button">See projects</button>
+        </div>
+      </section>
+      <section class="About">
+        <div class="Myself">
+          <h2>
+            About <br />
+            Myself
+          </h2>
+          <p>
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h3>LET'S CONNECT</h3>
+          <ul>
+            <a href="https://twitter.com/SolomonAfutu"
+              ><i class="fa fa-twitter fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href="https://www.linkedin.com/in/solomon-afutu-164b95255/"
+              ><i class="fa fa-linkedin fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href="https://github.com/SolomonTheCoder"
+              ><i class="fa fa-github fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href="">
+              <i class="fa fa-medium fa-2x" aria-hidden="true"></i
+            ></a>
+            <a href=""
+              ><i class="fa fa-hand-peace-o fa-2x" aria-hidden="true"></i
+            ></a>
+          </ul>
+          <button type="button">Get my resume</button>
+        </div>
+        <div class="Language">
+          <h2>Languages</h2>
```

Comment:
- Kindly add the arrow button as shown in the Figma design. your design must stick to the Figma design as mutch as possible :+1: 

![image](https://user-images.githubusercontent.com/95297251/227222049-dae562bb-c948-4f63-a24d-a04c31fbb101.png)